draft-ietf-dnsop-svcb-https.md

```diff
@@ -1048,8 +1052,9 @@ if a usable Alt-Svc value is available in the local cache.
 If Alt-Svc connection fails, these clients SHOULD fall back to the HTTPS RR
 client connection procedure ({{client-behavior}}).
 
-For clients that implement support for ECH, the interaction between
-HTTPS RRs and Alt-Svc is described in {{ech-client-behavior}}.
+Clients that implement support for ECH MUST perform the HTTPS RR query first,
+and MUST only make use of Alt-Svc when operating in SVCB-optional mode (see
+{{ech-client-behavior}}).
 
 This specification does not alter the DNS queries performed when connecting
 to an Alt-Svc hostname (typically A and/or AAAA only).
```

Comment:
Citation?

Reply:
Clarified

Reply:
I was noting that this might need citation, though the clarification is fine.